static/js/markdown.js

```diff
@@ -26,11 +26,11 @@ let helpers;
 
 // Regexes that match some of our common backend-only Markdown syntax
 const backend_only_markdown_re = [
-    // Inline image previews, check for contiguous chars ending in image suffix
+    // Inline image and video previews, check for contiguous chars ending in image and video suffix
     // To keep the below regexes simple, split them out for the end-of-message case
 
-    /\S*(?:\.bmp|\.gif|\.jpg|\.jpeg|\.png|\.webp)\)?\s+/m,
-    /\S*(?:\.bmp|\.gif|\.jpg|\.jpeg|\.png|\.webp)\)?$/m,
+    /\S*(?:\.bmp|\.gif|\.jpg|\.jpeg|\.png|\.webp|\.mp4|\.webm|\.mov|\.avi|\.ogg|\.wmv)\)?\s+/m,
```

Comment:
Where'd this list of extensions come from, out of curiosity?  On the server side, it looks like `.ogg` is guessed as audio, not video:

```py3
import mimetypes
for ext in ["mp4","webm","mov","avi","ogg","wmv"]:
    type = mimetypes.guess_type(f"https://chmrr.net/foo.{ext}")[0]
    print(f"{ext}: {type}")
```
```
mp4: video/mp4
webm: video/webm
mov: video/quicktime
avi: video/x-msvideo
ogg: audio/ogg
wmv: video/x-ms-wmv
```

So I think we should only be listing extensions which are guessed as `video/`, meaning dropping `ogg`

Reply:
Makes sense.